pyfar/classes/audio.py

```diff
@@ -735,12 +847,20 @@ def times(self):
     @property
     def frequencies(self):
         """Frequencies of the discrete signal spectrum."""
-        return np.atleast_1d(fft.rfftfreq(self.n_samples, self.sampling_rate))
+        if self.complex:
+            # assume the time domain data were complex-valued
+            # such that we need a two-sided Fourier spectrum
+
```

Comment:
```suggestion
```
I'd remove that line so it's clear that the comment refers to the line of code below and is not referring to something which was potentially deleted in the past